templates/crowdstrike_init_stack.yaml

```diff
@@ -460,7 +599,7 @@ Resources:
           - id: CKV_AWS_18
             comment: S3 access logs intentionally not enabled
     Properties:
-      BucketName: !Sub ${SourceS3BucketNamePrefix}-${AWS::AccountId}-${AWS::Region}
+      BucketName: !Sub ${SourceS3BucketNamePrefix}-${AWS::StackName}
```

Comment:
Using `AWS::StackName` results in S3 bucket creation to fail, when there are captial letters in the stackname. This is not scalable, please reset to `${AWS::AccountId}-${AWS::Region}`